packages/core/test/oneOf.test.jsx

```diff
@@ -817,13 +817,14 @@ describe('oneOf', () => {
 
     expect($select.value).eql('1');
 
-    sinon.assert.calledWithMatch(
-      onChange.lastCall,
-      {
-        formData: { ipsum: {}, lorem: undefined },
-      },
-      'root__oneof_select',
-    );
+    // After our fix, we no longer create unnecessary empty objects
```

Comment:
Should the same fix also apply to `anyOf` as well? IF so, can we get some updated tests for that too?

Reply:
This should apply to both and I will add coverage for `anyOf`, thanks!